config/jobs/kubernetes/sig-instrumentation/sig-instrumentation-kind-periodics.yaml

```diff
@@ -0,0 +1,59 @@
+periodics:
+- interval: 1h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-kind-e2e-structured-logging
+  annotations:
+    testgrid-dashboards: sig-instrumentation-tests, sig-testing-kind
+    testgrid-tab-name: kind-structured-logging
+    description: Smoke tests Kubelet with structured logging enabled using sigs.k8s.io/kind
+    # TODO: anyone else?
+    testgrid-alert-email: bentheelder@google.com
+    testgrid-num-columns-recent: '6'
+    fork-per-release: "true"
+    fork-per-release-periodic-interval: 1h 2h 6h 24h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/krte:v20210311-6badf7a-master
+      command:
+      - wrapper.sh
+      - bash
+      - -c
+      - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
+      env:
+      # this is the key configuration
+      - name: KUBELET_LOG_FORMAT
+        value: structured
+      # TODO: This is a fairly arbitrary test that actually schedules some pod(s).
+      # We mostly just want cluster up/down and something that ensures the cluster is in fact up fully.
```

Comment:
Just to clarify for my understanding,  we don't actually want to test what the logs are returning, we just want to make sure that the cluster is up and the kubelet does not complain

Reply:
Currently yes.
If we're interested in testing what the logs are returning we can but:
- it's not going to be portable to different kinds of clusters (seeing as kubelet logs are node level)
- we already know the logs are not fully migrated through static analysis (and can be pretty confident that they are eventually with it too)

Recently kubelet wouldn't even run with this enabled, so it's a start to make sure the cluster runs.